### ISSUE TITLE
100% CPU usage running Liquidator Bot

### ISSUE THREAD

**Issue by sebastianloves** [drift-labs/keeper-bots-v2#185]
The April 15th release of the Liquidator Bot seems to have an 100% CPU issue when its activated. Tried a few different computers and AWS servers and they all lead to 100% CPU usage on a single core. 

Please fix because liquidator is unusable.

**Comment by wphan** [comment]
Thanks for reporting this, we run instances of the liquidator and dont see 100% cpu usage, can you share some more details on how you are running the bot?

**Comment by sebastianloves** [comment]
I just git cloned the repository
and compiled using the Docker instructions.
Issue is the Usermap is like 60000 Users and it constantly checks if a user is liquidable and runs at 100% CPU usage. The one from March 2024 didn't do this. Only the one released after April 15th. I think a delay is missing somewhere in the code.

**Comment by wphan** [comment]
were you running the bot consistently? the number of user account has increased significantly in the last couple months, how many resources are you allocating to the docker container?

**Comment by sebastianloves** [comment]
Yes been running for a few weeks. I tried various hardware such as ram up to 8GB, platinum xeon processors from AWS. Tried under different platforms of Linux. Only one core works with the bot and that core is always 100%. How large was the userMap in March 2024 compared to April 2024?
